spec/wizards/claims/request_clawback_wizard_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe Claims::RequestClawbackWizard do
+  subject(:wizard) { described_class.new(claim:, state:, params:, current_step: nil) }
+
+  let(:state) { {} }
+  let(:params_data) { {} }
+  let(:params) { ActionController::Parameters.new(params_data) }
+  let(:claim) { create(:claim, status: "sampling_in_progress") }
+
+  before do
+    allow(claim).to receive(:save!).and_return(true)
+  end
+
+  describe "#steps" do
+    subject { wizard.steps.keys }
+
+    it { is_expected.to eq %i[clawback check_your_answers] }
+  end
+
+  describe "#update_status" do
+    it "updates the claim status to 'clawback_requested' and saves the claim" do
+      wizard.update_status
+      expect(claim.status).to eq("clawback_requested")
+      expect(claim).to have_received(:save!)
+    end
```

Comment:
Could we test the unhappy paths here?

Reply:
@Kizr I haven't yet added error handling (in anticipation of the new designs!) so I'll add to-dos for the following unhappy paths, can you think of any I might have missed?
- Support user tries to request a clawback of more hours than were claimed
- Support user tries to request a clawback of more hours than are available to clawback
- Support user tries to request a clawback outside of sampling/clawback period?

Reply:
They sounds sensible!